course/lesson_13/README.md

```diff
@@ -1,5 +1,3 @@
-### Lesson 13 - Debugging
+### Lesson 13 - Decorators + generators
```

Comment:
masz pomysł na sczegóły scopu tych tematów? jestem raczej na nie jeśli chodzi o implementację swoich dekoratorów, może lepiej dać generatory do list i zrobić 2 lekcje z obiektówki, wprowadzić classmethod i przy okazji tego omówić dekoratory

Reply:
Też jestem na nie jeśli chodzi o implementacje własnych dekoratorów. Bardziej miałem w głowie użycie gotowych i zrozumienie samej idei (wszystko jest obiektem etc.) Pomysł z połączeniem list i generatorów też ok.

Reply:
Ja nie wiem po co w ogóle generatory. Nie spodziewałbym się, że ktoś ogarnie korutyny na tym kursie i asyncio. Moim zdaniem dać najwyżej hinta, że zamiast [] może być () i jak ktoś chce niech czyta o generatorach. Ja się nie czuję na siłach wytłumaczyć tego komuś, kto się uczy programować, widziałem jak ludzie nie łapią co to zmienna... moja wiara jest ograniczona.

Reply:
Tez jestem na nie jeśli chodzi o pisanie własnych dekoratorów i poprzestał bym na użyciu gotowych.
I również bym się wstrzymał na tym poziomie z generatorami.

Reply:
Ok, generatory jeszcze do dyskusji.
IMO skoro mamy pythona 3.6 to dobrze byłoby poruszyć ten temat w najprostszej formie.

Reply:
Też nie jestem za wprowadzeniem generatorów na tym poziomie

Reply:
jestem za przesunięciem generatorów na następny etap